addons/mass_mailing/models/mailing.py

```diff
@@ -796,7 +796,7 @@ def _fetch_trace_res_ids(trace_domain):
     def action_view_mailing_contacts(self):
```

Comment:
Method name to update then, the purpose is not to view them but to add / import them.